app/auth/templates/auth/forgot_password.html

```diff
@@ -13,11 +13,11 @@
     {{ form.csrf_token }}
     <div class="card-body p-6">
       <div class="card-title">Forgot password</div>
-      <p class="text-muted">Enter your email address and your will receive an email to reset your password.</p>
+      <p class="text-muted">Enter your email address you signed up with and you will receive an email to reset your password.</p>
```

Comment:
I think we can remove this phrase completely as it's quite obvious given the placeholder.